### DISCUSSION TITLE
How to interact with PHP?

### DISCUSSION THREAD

**Discussion by Tok1shu** [plan-player-analytics/Plan#2740] [APIs and Development]
Good evening, I have a question, I calculate the played time through time in php, but I think it’s wrong, can this be fixed?
![image](https://user-images.githubusercontent.com/111585037/202873976-2ee85dc6-4f54-49aa-a381-f85e52eb5672.png)
![image](https://user-images.githubusercontent.com/111585037/202873982-28e4a10f-9672-4d4d-a7f0-fa180937179b.png)
![image](https://user-images.githubusercontent.com/111585037/202873988-53f3e692-ad47-406e-971b-ba32efbde2a9.png)

**Comment by AuroraLS3** [comment]
Time in Plan database is stored in milliseconds.

You are getting the register date instead of playtime.

Playtime is calculated from plan_sessions table.

```sql
SELECT uuid, SUM(session_end-session_start) as playtime 
FROM plan_sessions s
JOIN plan_users u on u.id=s.user_id
GROUP BY uuid
```

**Comment by AuroraLS3** [comment]
You can also use the json endpoints (Documented at `<plan webserver>/docs`)
